cmd/get/get.go

```diff
@@ -57,3 +153,101 @@ func toPrintableProjects(projects []*admin.Project) []interface{} {
 
 	return res
 }
+
+func toPrintableTask(tasks []*admin.NamedEntityIdentifier) []interface{} {
```

Comment:
I guess this is what we want to auto-generate, but until then we should add tests?

Reply:
ok